torch/fx/experimental/symbolic_shapes.py

```diff
@@ -2750,6 +2750,9 @@ def create_symbol(
             symbolic_context.shape_env_to_source_to_symbol_cache[id(self)][source_name] = r
         return r
 
+    def create_named_symbol_with_val(self, expr: sympy.Symbol, val: int):
```

Comment:
This seems like a confusing name, since there are existing APIs like `create_symbol`. Maybe something explicit would be better, like `add_var_to_val`?

Also, should you not check whether the symbol already exists? (Probably doesn't matter but still...)

Reply:
fixed.